beater/server.go

```diff
@@ -1,48 +1,34 @@
-package server
+package beater
 
 import (
 	"compress/gzip"
 	"compress/zlib"
 	"context"
 	"crypto/subtle"
 	"encoding/json"
-	"flag"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"net/http"
 	"strings"
 
+	"flag"
```

Comment:
@jalvz Could you clean up the imports here in the follow up PR https://github.com/elastic/apm-server/pull/85